build/NuSpecs/UmbracoCms.nuspec

```
@@ -30,6 +30,7 @@
         -->
         <contentFiles />
     </metadata>
-    <files>
-    </files>
+  <files>
+    <file src="buildTransitive\**" target="buildTransitive\" />
```

We can make this a bit more specific:
```suggestion
    <file src="buildTransitive\Umbraco.Cms.props" target="buildTransitive" />
```